src/content/byoc/gcp/index.mdx

```diff
@@ -27,7 +27,7 @@ Once the project is created, contact your Okteto representative and share the fo
 
 ## Step 2: Grant Access
 
-Your sales rep will provide a service account that Okteto uses to access your project. Grant that service account the **Editor** role in the project you just created:
+Your sales rep will provide a service account that Okteto uses to access your project. Grant that service account the **Admin** role in the project you just created:
```

Comment:
could you also update it here?
https://github.com/okteto/docs/blob/main/versioned_docs/version-1.33/byoc/gcp/index.mdx